### DISCUSSION TITLE
https://blog.eledah.ir/projects/pkm/%D8%A7%D8%B2-%DB%8C%D8%A7%D8%AF%D8%AF%D8%A7%D8%B4%D8%AA-%D8%A8%D9%87-%D8%B3%D8%A7%DB%8C%D8%AA-%D8%A8%D8%A7-%DA%A9%D9%88%D8%A7%D8%B1%D8%AA%D8%B2

### DISCUSSION THREAD

**Discussion by giscus** [eledah/quartz_blog#20] [Announcements]
# https://blog.eledah.ir/%D9%BE%D8%B1%D9%88%DA%98%D9%87%E2%80%8C%D9%87%D8%A7/%DB%8C%D8%A7%D8%AF%D8%AF%D8%A7%D8%B4%D8%AA%E2%80%8C%D8%A8%D8%B1%D8%AF%D8%A7%D8%B1%DB%8C/%D8%A7%D8%B2-%DB%8C%D8%A7%D8%AF%D8%AF%D8%A7%D8%B4%D8%AA-%D8%A8%D9%87-%D8%B3%D8%A7%DB%8C%D8%AA-%D8%A8%D8%A7-%DA%A9%D9%88%D8%A7%D8%B1%D8%AA%D8%B2

کوارتز چیه؟ کوارتز یک Static Site Generator رایگان است. سایتی که الان جلوی روی شماست، با کوارتز ساخته شده. به زبان ساده، کوارتز یادداشت‌های ابسیدین را به صورت ورودی می‌گیرد و به عنوان خروجی، یک سایت تحویل می‌دهد.

https://blog.eledah.ir/%D9%BE%D8%B1%D9%88%DA%98%D9%87%E2%80%8C%D9%87%D8%A7/%DB%8C%D8%A7%D8%AF%D8%AF%D8%A7%D8%B4%D8%AA%E2%80%8C%D8%A8%D8%B1%D8%AF%D8%A7%D8%B1%DB%8C/%D8%A7%D8%B2-%DB%8C%D8%A7%D8%AF%D8%AF%D8%A7%D8%B4%D8%AA-%D8%A8%D9%87-%D8%B3%D8%A7%DB%8C%D8%AA-%D8%A8%D8%A7-%DA%A9%D9%88%D8%A7%D8%B1%D8%AA%D8%B2

**Comment by rahmadi89** [comment]
ممنون بابت مطلب خوبتون
جسارتا شما میدونین دلیل اینکه زبان قسمت کامنت های من انگلیسیه چیه ؟
RouhollahAhmadi.ir

**Reply by rahmadi89** [reply]
https://github.com/rahmadi89/Site

**Reply by eledah** [reply]
سلام 😄
توی فایل

```
quartz/components/Comments.tsx
```

تنظیمات بخش کامنت انجام می‌شه. اونجا باید بهش بفهمونی که از زبان فارسی استفاده کنه. برای این کار، توی همون فایل `const theme =document.document...` رو پیدا کن و خط زیر رو پایینش اضافه کن.

```
giscusScript.setAttribute("data-lang", "fa")
```

توی خود مطلب هم اضافه کردم. ممنون بابت گفتنش.

**Comment by Logical-Developer** [comment]
ممنون، آموزش جامعی تهیه کردید.
درخصوص قالب برای سایت راهنمایی کنید، شما این قالب را از کجا دریافت کردید ؟
لطفا برای دریافت قالب هم راهنمایی بفرمایید.

درغیر اینصورت تا چند ماه دیگه تمام سایت های فارسی زبان یک شکل میشن ;)     :))

**Reply by eledah** [reply]
سلام. من قالب اصلی رو یه مقداری دستکاری کردم. سورس تغییرات هم روی گیت‌هابم هست. اگر با HTML/CSS/TS آشنا هستید، راحت می‌شه توی سایت تغییرات ایجاد کرد. مثلاً سایت [سوکراتیکا](https://toolbox.socratica.info/) که اول متن هم اومده، با کوارتز ساخته شده. کلی سایت دیگه هم هست که [اینجا](https://quartz.jzhao.xyz/showcase) اومده و قابلیت‌های کوارتز رو نشون می‌ده.
